pykern/pkcli/test.py

```diff
@@ -94,33 +99,47 @@ def default_command(*args):
 def _args(tests):
     paths = []
     flags = []
+    s = None
     for t in tests:
         if "=" in t:
             a, b = t.split("=")
-            if a == "case":
+            if len(b) <= 0:
+                pykern.pkcli.command_error(f"empty value for option={t}")
+            elif a == "case":
                 flags.extend(("-k", b))
+            elif a == "skip_past":
+                s = b
             else:
-                pykern.pkcli.command_error("unsupported option={}".format(t))
+                pykern.pkcli.command_error(f"unsupported option={t}")
         else:
             paths.append(t)
-    return _find(paths), flags
+    return _find(paths, s), flags
 
 
-def _find(paths):
+def _find(paths, skip_past):
     from pykern import pkio
 
-    i = re.compile(r"(?:_work|_data)/")
     res = []
+
+    def _file(path):
+        nonlocal skip_past
+        if skip_past:
+            if skip_past in path:
+                skip_past = None
+            return
+        res.append(path)
```

Comment:
I might be missing something here, but it seems like this is saying, if the skip path is in the path, dont append it, otherwise append it. But how are we assuring all files up to skip path are being skipped as well?

Reply:
Walk through the logic again including line 126. 128 only is hit after all the files have been skipped before skip_past matches.

Reply:
Oh, gotcha